guides/doc-Planning_for_Project/topics/Required_Technical_Users.adoc

```diff
@@ -25,14 +25,20 @@ Because of potential conflicts with local users that {Project} creates, you cann
 |User name |UID |Group name |GID |Flexible UID and GID |Home |Shell
 |foreman |N/A |foreman |N/A |yes |/usr/share/foreman |/sbin/nologin
 |foreman-proxy |N/A |foreman-proxy |N/A |yes |/usr/share/foreman-proxy |/sbin/nologin
+|apache |48 |apache |48 |no |/usr/share/httpd |/sbin/nologin
+|postgres |26 |postgres |26 |no |/var/lib/pgsql |/bin/bash
+ifdef::katello,orcharhino,satellite[]
 |pulp |N/A |pulp |N/A |no |N/A |/sbin/nologin
+endif::[]
 |puppet |N/A |puppet |N/A |yes |/opt/puppetlabs/server/data/puppetserver |/sbin/nologin
 |puppetserver |N/A |puppet |N/A |no |/opt/puppetlabs/server/data/puppetserver |/sbin/nologin
-|qdrouterd |N/A |qdrouterd |N/A |yes |N/A |/sbin/nologin
+ifdef::katello,orcharhino,satellite[]
+|qdrouterd |N/A |qdrouterd |N/A |yes |/var/lib/qdrouterd |/sbin/nologin
 |qpidd |N/A |qpidd |N/A |yes |/var/lib/qpidd |/sbin/nologin
-|unbound |N/A |unbound |N/A |yes |/etc/unbound |/sbin/nologin
-|postgres |26 |postgres |26 |no |/var/lib/pgsql |/bin/bash
-|apache |48 |apache |48 |no |/usr/share/httpd |/sbin/nologin
+|saslauth |N/A |saslauth |76 |no |/run/saslauthd |/sbin/nologin
 |tomcat |53 |tomcat |53 |no |/usr/share/tomcat |/bin/nologin
-|saslauth |N/A |saslauth |76 |no |N/A |/sbin/nologin
+endif::[]
+ifndef::foreman-deb[]
+|unbound |N/A |unbound |N/A |yes |/etc/unbound |/sbin/nologin
```

Comment:
it gets created by the `unbound-libs` package, but I don't see anything pulling that package in (it is in the base EL install, tho)

Reply:
It's not in a container and at least Foreman doesn't depend on it (I can use `dnf remove unbound-libs` on my server) so should we really list this?

Reply:
Ah, I didn't notice you just moved it around.